modules/health_quest/app/services/health_quest/patient_generated_data/common/identity_meta_info.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module PatientGeneratedData
+    module Common
+      module IdentityMetaInfo
+        ##
+        # The namespace URI for the FHIR::DSTU2::Identifier object
+        #
+        SYSTEM_ID = 'urn:uuid:2.16.840.1.113883.4.349'
+        ##
+        # Identity of the terminology system for the FHIR::DSTU2::Identifier object
+        #
+        CODING_SYSTEM = 'https://pki.dmdc.osd.mil/milconnect'
+        ##
+        # Operation Resource meta tag for the FHIR::DSTU2::Meta object
+        #   VA URI
+        META_SYSTEM = 'https://wiki.mobilehealth.va.gov/display/PGDMS/Client+Provenance+Mapping'
+        ##
+        # Operation Resource meta tag for the FHIR::DSTU2::Meta object
+        #   VA identifier
+        META_CODE = 'vagov-a0e116eb-faa1-4703-aafe-1a270128607a'
```

Comment:
Just checking if the values stores in the above constants are something that would be better suited for credstash?

Reply:
That's a really good idea! I'd like to wait for a little bit until I've implemented the upcoming Questionnaire resource story before I proceed with such a refactor, using its own PR.

Reply:
Ok great, definitely a good refactor for a future PR